simple-page-ordering.php

```diff
@@ -186,22 +192,27 @@ public static function ajax_simple_page_ordering() {
 					'menu_order' => 'ASC',
 					'title'      => 'ASC',
 				),
-				'post__not_in'           => $excluded,
 				'update_post_term_cache' => false,
 				'update_post_meta_cache' => false,
-				'suppress_filters'       => true,
+				'suppress_filters'       => true, // phpcs:ignore WordPressVIPMinimum.Performance.WPQueryParams.SuppressFiltersTrue
```

Comment:
Set this to ignore the rule because I am pretty sure that we are ignoring caching while performing this query.